v3/docs/03-runtime/m-upgrades/index.mdx

```diff
@@ -47,35 +47,30 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
-  this; it serves only as an indication that the code is different; as long as the other two
-  versions are the same then while the actual code may be different, it is nonetheless required to
-  do the same thing. Non-consensus-breaking optimizations are about the only changes that could be
-  made which would result in only the `impl_version` changing.
-
-- `transaction_version`: The version of the extrinsics interface. This number must be updated in the
-  following circumstances: extrinsic parameters (number, order, or types) have been changed;
-  extrinsics or pallets have been removed; or the pallet order in the `construct_runtime!` macro
-  _or_ extrinsic order in a pallet has been changed. If this number is updated, then the
-  `spec_version` must also be updated.
-
-- `apis` is a list of supported
-  [runtime APIs](/rustdocs/latest/sp_api/macro.impl_runtime_apis.html) along
-  with their versions.
-
-As mentioned above, the executor always verifies that the native runtime has the same
-consensus-driven logic before it chooses to execute it, independent of whether the version is higher
-or lower.
+- `impl_name`: The name of the implementation of the spec. 
+  This is of little consequence for the node and serves only to differentiate code of different implementation teams.
+
+- `authoring_version`: The version of the authorship interface. 
+  An authoring node will not attempt to author blocks unless this is equal to its native runtime.
+
+- `spec_version`: The version of the runtime specification. 
+  A full node will not attempt to use its native runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`, `spec_version`, and `authoring_version` are the same between Wasm and native. 
+  This should always get bumped whenever there is a new `transaction_version` bump.
+
+- `impl_version`: The version of the implementation of the specification. 
+  Nodes are free to ignore this; it serves only as an indication that the code is different. 
+  As long as the other two versions are the same then while the actual code may be different, it is required to do the same thing. 
+  Non-consensus-breaking optimizations are about the only changes that could be made which would result in only the `impl_version` changing.
+
+- `transaction_version`: The version of the extrinsics interface, useful to synchronize firmware updates for hardware wallets. 
+  This allows hardware wallets to know which transactions they can safely sign.
+  This number must be bumped when there is a change in the order of the pallets in the `construct_runtime!` macro or when there is a change in the dispatchables of a pallet.
+  This includes any change in pallet and dispatchable runtime parameter types. 
+  If this number is updated, then the `spec_version` must also be updated which can be automated as a CI process, see [this example](https://gitlab.parity.io/parity/polkadot/-/blob/master/scripts/gitlab/check_extrinsics_ordering.sh).
```

Comment:
```suggestion
  If this number is updated, then the `spec_version` must also be updated. Updates to the `spec_version` can be automated as a CI process, see [this example](https://gitlab.parity.io/parity/polkadot/-/blob/master/scripts/gitlab/check_extrinsics_ordering.sh).
```
It's best to avoid these types of generic [this example] link labels.